### PR TITLE
SBS-4 sbs-4

### PR DESCRIPTION
**Pull request by mpaz-redstage**

### CODE REVIEW

File: 1.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+sdf
+asdf
```
Comment (mpaz-redstage) [inline 1.txt:2]: test

Comment (mpaz-redstage) [reply 1.txt:2]: test

Comment (mpaz-redstage) [reply 1.txt:2]: test

Comment (mauriciopazpp) [reply 1.txt:2]: test

**Review by mauriciopazpp — CHANGES_REQUESTED** [review]
!

**Review by mauriciopazpp — APPROVED** [review]
w

File: 1.txt (hunk 1 above)
Comment (mpaz-redstage) [reply 1.txt:2]: test

**Review by mpaz-redstage — COMMENTED** [review]
test

**Comment by mpaz-redstage** [comment]
test

**Comment by mpaz-redstage** [comment]
asdfasf

File: 1.txt (hunk 1 above)
Comment (mpaz-redstage) [reply 1.txt:2]: test

**Pull request closed without merging**